app_archivist.md

```diff
@@ -0,0 +1,14 @@
+# <img src="/images/yunohost_package.png" height="80px" alt="Package"> Archivist
+
+[![Install archivist with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=archivist) [![Integration level](https://dash.yunohost.org/integration/archivist.svg)](https://dash.yunohost.org/appci/app/archivist)
+
+- [Useful links](#useful-links)
+
+Archivist is an automatic backup system for your server. It able to makes backups of your YunoHost core and your apps by using the YunoHost backup command. It can also makes backups of specified directories.
```

Comment:
```suggestion
Archivist is an automatic backup system for your server. It is able to makes backups of your YunoHost core and your apps by using the YunoHost backup command. It can also makes backups of specified directories.
```